content/06.conclusion_future_direction.md

```diff
@@ -1,7 +1,13 @@
-# Conclusion and Future Direction
-Recap the original problem - takes a long time to create useful label function
+## Conclusion and Future Direction
 
-Proposed solution - reuse label functions
-
-Mention incorporating more relationships
-Mention creating a centralized multitask text extractor using this method.
+Filling out knowledge bases via manual curation can be an arduous and erroneous task [@doi:10.1093/bioinformatics/btm229].
+As the rate of publications increases manual curation becomes an infeasible approach.
+Data programming, a paradigm that uses label functions a means to speed up the annotation process, can be used as a solution for this problem.
+A problem with this paradigm is creating a useful label function takes significant amount of time.
+We tested the feasibility of reusing label functions as a way to speed up the process of creating label functions.
+Based on our findings, we conclude that label functions can be reused across edge types.
+Adding more relavant label functions can increase overall performance.
+The discriminative model, under this paradigm, has a tendency to overfit to predictions of the generative model.
+We recommend implementing regularization techniques such as drop out and weight decay to combat this issue.
+Lastly, this work lays the foundation for creating a robust multi-relationship extractor (aka multitask learning [@doi:10.1371/journal.pone.0200699]).
+Through this system we can quickly scale Hetionet and provide researchers with quick access to new findings and publications.
```

Comment:
Would be good to briefly touch on why this would be a huge advance. This could also go in intro, but would be a strong ending, so up to here. Here are my notes on the benefits of constructing hetnets using NLP / text mining:

- Common framework for all edge types
- Continuous quality scores for each edge
- Can release openly: free of upstream licensing restrictions
- Can run periodically to ingest new knowledge, automatically
- Provenanced edges: users can go from edges to papers/sentences supporting the assertion